tests/integration/test_fim/test_registry/test_recursion_level/test_recursion_level_registry.py

```diff
@@ -22,152 +19,134 @@
 key = "HKEY_LOCAL_MACHINE"
 registry = "SOFTWARE"
 registry_no_recursion = os.path.join(registry, 'norecursion')
-registry_recrusion_1 = os.path.join(registry, 'recursion1')
-registry_recrusion_5 = os.path.join(registry, 'recursion5')
-registry_recrusion_512 = os.path.join(registry, 'recursion512')
+registry_recursion_1 = os.path.join(registry, 'recursion1')
+registry_recursion_5 = os.path.join(registry, 'recursion5')
+registry_recursion_29 = os.path.join(registry, 'recursion29')
+
+test_regs = [os.path.join(key, registry_no_recursion),
+             os.path.join(key, registry_recursion_1),
+             os.path.join(key, registry_recursion_5),
+             os.path.join(key, registry_recursion_29)]
+
+rl_dict = {
+    registry_no_recursion: '0',
+    registry_recursion_1: '1',
+    registry_recursion_5: '5',
+    registry_recursion_29: '29'
+}
+
+conf_params = {'REGISTRY_0': os.path.join(key, registry_no_recursion),
+               'LEVEL_0': rl_dict[registry_no_recursion],
+
+               'REGISTRY_1': os.path.join(key, registry_recursion_1),
+               'LEVEL_1': rl_dict[registry_recursion_1],
+
+               'REGISTRY_2': os.path.join(key, registry_recursion_5),
+               'LEVEL_2': rl_dict[registry_recursion_5],
+
+               'REGISTRY_3': os.path.join(key, registry_recursion_29),
+               'LEVEL_3': rl_dict[registry_recursion_29]
+}
```

Comment:
This closing bracket does not match visual indentation.
```suggestion
               }
```